modules/xmpp/ChatRoom.js

```diff
@@ -102,13 +105,11 @@ function tryParseJSONAndVerify(jsonString) {
         // so we must check for that, too.
         // Thankfully, null is falsey, so this suffices:
         if (json && typeof json === 'object') {
-            const topic = json[VALID_FIELD_NAMES[0]];
-            const payload = json[VALID_FIELD_NAMES[1]];
+            const topic = json[JITSI_MEET_MUC_TOPIC];
```

Comment:
why do we need this muc topic thing again?  i thought we were going to use a different element?

Reply:
Eventually we will. But I'm not doing the change right now, because we need to send these from jigasi :(

Reply:
ok...but why does sending them from jigasi affect it?

Reply:
Because jigasi can only send a jitsi Message, and can not add an xml extension to it...

Reply:
Ah, didn't realize it was sending it through the web.  Bummer.  So will there be a way we can get around that in the future?

Reply:
As soon as we figure out (and implement) a way to make jigasi send the extension